src/LazySet.jl

```diff
@@ -341,3 +342,50 @@ list of linear constraints.
 This fallback implementation relies on `constraints_list(S)`.
 """
 tosimplehrep(S::LazySet) = tosimplehrep(constraints_list(S))
+
+"""
+    isuniversal(X::LazySet{N}, [witness]::Bool=false
+               )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+
+Check whether a given convex set is universal, and otherwise optionally compute
+a witness.
+
+### Input
+
+- `X`       -- convex set
+- `witness` -- (optional, default: `false`) compute a witness if activated
+
+### Output
+
+* If `witness` option is deactivated: `true` iff ``X`` is universal
+* If `witness` option is activated:
+  * `(true, [])` iff ``X`` is universal
+  * `(false, v)` iff ``X`` is not universal and ``v ∉ X``
+
+### Notes
+
+This is a naive fallback implementation.
+"""
+function isuniversal(X::LazySet{N}, witness::Bool=false
```

Comment:
I think that it's better to use dispatch for `isuniversal` and do the appropriate tests on each type (or abstract type).

Reply:
Yes, I wanted to create a follow-up issue for this, but forgot about it. I added #1125.